website/static/js/pages/render-nodes.js

```diff
@@ -2,9 +2,12 @@
 
 var $osf = require('js/osfHelpers');
 var $ = require('jquery');
+var ko = require('knockout');
 
 // model for components, due to simplicity did not create a new file
-var ComponentControl = {};
+var ComponentControl = {
+     anonymousUserName:ko.observable('<em>A user</em>')
```

Comment:
@sloria Yes, it is static content, but the thing is that if the content is put in html, it will show before the knockout content loads completely, which is the reason why "A User" show.

This is one way to solve it. Another way is to use knockout observable variable to control the visibility of the html.